tests/filecheck/transforms/convert_qssa_to_qref.mlir

```diff
@@ -3,23 +3,23 @@
 
 %q0, %q1 = qssa.alloc<2>
 %q2 = qssa.h %q0
-%q3, %q4 = qssa.cz %q1, %q2
-%q5, %q6 = qssa.cnot %q3, %q4
-%0 = qssa.measure %q6
+%q3 = qssa.rz <pi/2> %q1
+%q4, %q5 = qssa.cnot %q2, %q3
+%0 = qssa.measure %q4
 
 // CHECK:       builtin.module {
 // CHECK-NEXT:    %q0, %q1 = qref.alloc<2>
 // CHECK-NEXT:    qref.h %q0
-// CHECK-NEXT:    qref.cz %q1, %q0
-// CHECK-NEXT:    qref.cnot %q1, %q0
+// CHECK-NEXT:    qref.rz <pi/2> %q1
```

Comment:
@PapyChacal do you know why the attribute name is not printed here?

Reply:
see https://github.com/xdslproject/xdsl/issues/2807